.github/workflows/ci-goreleaser.yaml

```diff
@@ -35,7 +35,8 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19.1
+          go-version: '1.19'
+          check-latest: true
```

Comment:
Isn't this being bumped by dependabot already when new versions are available? What happens here if there's a 1.20 available?

Reply:
dependabot does not bump this part, it only bumps the action itself
if 1.20 is out we need to update and push a PR

Reply:
Right, dependabot will only change that part. When 1.20 is out, will we get build failures, warnings, or things like that?

Reply:
no, we will still run go1.19 only if we upgrade to go1.20

Reply:
Thanks for the clarification!

Reply:
thanks for the review